recipe/meta.yaml

```diff
@@ -25,19 +25,18 @@ requirements:
     - setuptools
     - wheel
     # These are also needed at build time.
-    - bleach 4.1.0
-    - bokeh 3.3.0
-    - pyct 0.5.0
-    - param 2.0
-    - pyviz_comms
-    - nodejs 18.16.0
-    - requests 2.31.0
-    - tqdm 4.65.0
+    - bokeh >=3.2.0,<3.4
+    - pyct >=0.4.4
+    - param >=2.0.0,<3
```

Comment:
```suggestion
    - param >=2.0.0
```

Reply:
Being conservative, I am leaving the upper bound